spec/ripple/conflict/resolver_spec.rb

```diff
@@ -8,10 +8,10 @@ module Conflict
       end
 
       describe 'calling the lambda returned by .to_proc' do
-        let(:sibling_1) { stub(:data => { '_type' => 'User' } ) }
-        let(:sibling_2) { stub(:data => { '_type' => 'User' } ) }
+        let(:sibling_1) { stub(:data => { '_type' => 'User' }, :content_type => 'application/json') }
```

Comment:
Hmm.. The content type was added to get the two existing resolver specs to pass since we expect objects to respond to `content_type`. All `RObject` instances respond to `content_type` so I believe this should be a safe change.